app/views/posts/index.html.erb

```diff
@@ -1,11 +1,11 @@
-<h1>posts#index</h1>
-
-<ul>
-  <li>User name <span>number of posts</span></li>
-  <li>
-    <h2>Bio</h2>
-    <p>Lorem ipsum amet</p>
-  </li>
-  <li>Post 1</li>
-  <li>Comments</li>
-</ul>
\ No newline at end of file
+<div>
+  <%= render 'users/user', user: @post.user %>
+  <ul>
+    <%@post.user.posts.each do |post| %>
```

Comment:
- [ ] <b>REQUIRED:</b> Considering you have addressed the previous issues as I suggested, now go on and use it by simply `@posts.each do |post| ` instead of `@post.user.posts.each do |post|`

Reply:
Improved 👍

Reply:
Thank you for addressing these two issues 💯 👍